docs/templates/plugin.rst.j2

```diff
@@ -190,8 +190,6 @@ Parameters
             {% if value.suboptions %}
                 {% if value.suboptions.items %}
                     @{ loop(value.suboptions|dictsort) }@
-                {% elif value.suboptions[0].items %}
-                    @{ loop(value.suboptions[0]|dictsort) }@
```

Comment:
Isn't this still needed, assuming the
```
elif isinstance(v['suboptions'][0], dict):
                    v['suboptions'] = process_options(module, v['suboptions'][0])
```
code in the other file has relevance?

I'll note that I'm not familiar (even slightly) with the plugin.rst.j2 stuff, though.

Reply:
If these lines would have stayed, it wouldn't be needed, but it turned out that the return value was not what I expected. I've re-added these lines.